src/main/java/com/mjbmjb/cf/taskmaster/taskmaster/controller/TaskController.java

```diff
@@ -27,11 +27,13 @@ String getRootPath() {
         return "Hello, from the world of tomorrow!";
     }
 
+    @CrossOrigin
```

Comment:
You can put this annotation above the class definition and it'll work on all routes